docs/your_first_grid.md

```diff
@@ -58,6 +60,80 @@ sylius_grid:
                         template: SyliusUiBundle:Grid/Field:enabled.html.twig # This will be a checkbox field
 ```
 
+</details>
+
+<details open><summary>PHP</summary>
+
+```php
+<?php
+
+use App\Entity\Suplier;
+use Sylius\Bundle\GridBundle\Builder\GridBuilder;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\Field\TwigField;
+use Sylius\Bundle\GridBundle\Config\GridConfig;
+
+return static function (GridConfig $grid) {
+    $grid->addGrid(GridBuilder::create('app_admin_supplier', Suplier::class)
+        ->addField(
+            StringField::create('name')
+                ->setLabel('sylius.ui.name')
+        )
+        ->addField(
+            TwigField::create('enabled', 'SyliusUiBundle:Grid/Field:enabled.html.twig')
+                ->setLabel('sylius.ui.enabled')
+        )
+    )
+};
+```
+
+OR
+
+```php
+<?php
+# src/Grid/AdminSupplierGrid.php
+
+declare(strict_types=1);
+
+namespace App\Grid;
+
+use App\Entity\Suplier;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\Field\TwigField;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+final class AdminSupplierGrid extends AbstractGrid implements ResourceAwareGridInterface
```

Comment:
Maybe just `AdminSupplier`? Here and in other places. As we're using also `app_admin_supplier` in yaml instead of `app_admin_supplier_grid`? It's already placed in the `Grid` namespace and extends the `Abstract**Grid**` class. But I have no _strong_ opinion about this 🖖